src/Simulation/TargetDefinitions/Decompositions/ApplyControlledX.qs

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Quantum.Intrinsic {
+
+    /// # Summary
+    /// Applies the controlled-X (or CNOT) gate to a pair of qubits. Does not support 
+    /// the Controlled functor.
+    ///
+    /// # Description
+    /// \begin{align}
+    ///     \operatorname{CNOT} \mathrel{:=}
+    ///     \begin{bmatrix}
+    ///         1 & 0 & 0 & 0 \\\\
+    ///         0 & 1 & 0 & 0 \\\\
+    ///         0 & 0 & 0 & 1 \\\\
+    ///         0 & 0 & 1 & 0
+    ///     \end{bmatrix},
+    /// \end{align}
+    ///
+    /// where rows and columns are ordered as in the quantum concepts guide.
```

Comment:
It probably makes sense to make the fragment `quantum concepts` a web-link, if applicable and simple.

Reply:
Unfortunately I don't have link for this, and many of our intrinsic operations list the same text without a link.